src/pages/computer-science/data-structures/trees/index.md

```diff
@@ -12,4 +12,23 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
+A binary tree is a non linear data structure consisting nodes, where each node has the following 3 components:
```

Comment:
Hello, You need to replace the stub article with your new article and it would be great if you can add more details on this topic (optional)

Please read this style guide: https://github.com/freeCodeCamp/guides#article-style-guide

Please let me know if you have any questions.